corporate/views/plan_activity.py

```diff
@@ -0,0 +1,64 @@
+from typing import Any
+
+from django.http import HttpRequest, HttpResponse
+from django.shortcuts import render
+
+from corporate.lib.activity import ActivityHeaderEntry, format_optional_datetime, make_table
+from corporate.models import Customer, CustomerPlan, LicenseLedger
+from zerver.decorator import require_server_admin
+
+
+def get_plan_billing_entity_name(customer: Customer) -> str:
+    if customer.realm:
+        return customer.realm.name
+    elif customer.remote_realm:
+        return customer.remote_realm.name
+    assert customer.remote_server is not None
+    return customer.remote_server.hostname
```

Comment:
We might find we prefer the `realm.host` instead, but I guess we can see how it works in practice.